lib/chef/knife/vsphere_datastore_list.rb

```diff
@@ -18,23 +18,6 @@
 require 'chef/knife'
 require 'chef/knife/base_vsphere_command'
 
-def number_to_human_size(number)
```

Comment:
just added the ` number_to_human_size` to the base class, I am still unsure of how this worked if it was never in the base class.